tests/org.jboss.tools.central.ui.bot.test/src/org/jboss/tools/central/test/ui/reddeer/LauncherApplicationTest.java

```diff
@@ -0,0 +1,243 @@
+import org.jboss.tools.central.reddeer.projects.ArchetypeProject;
+import org.jboss.tools.common.launcher.reddeer.wizards.JBossCentralProjectWizardLauncherApplication;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ *
+ * @author zcervink
+ *
+ */
+@RunWith(RedDeerSuite.class)
+public class LauncherApplicationTest {
+
+	private static final String CENTRAL_LABEL = "Red Hat Central";
+	private static final String MAVEN_SETTINGS_PATH = System.getProperty("maven.config.file") == null
+			? "./target/classes/settings.xml"
+			: System.getProperty("maven.config.file");;
+
+	private static final Logger log = Logger.getLogger(ExamplesOperator.class);
+	private static final String BLACKLIST_ERRORS_REGEXES_FILE = "resources/blacklist-launcher-application-errors-regexes.json";
+	private static JSONObject blacklistErrorsFileContents;
+
+	@BeforeClass
+	public static void setup() {
+		String mvnConfigFileName = new File(MAVEN_SETTINGS_PATH).getAbsolutePath();
+		WorkbenchPreferenceDialog preferenceDialog = new WorkbenchPreferenceDialog();
+		preferenceDialog.open();
+		MavenSettingsPreferencePage prefPage = new MavenSettingsPreferencePage(preferenceDialog);
+		preferenceDialog.select(prefPage);
+		prefPage.setUserSettingsLocation(mvnConfigFileName);
+		preferenceDialog.ok();
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	@After
+	public void teardown() {
+		WorkbenchShellHandler.getInstance().closeAllNonWorbenchShells();
+		for (Project p : new ProjectExplorer().getProjects()) {
+			p.delete(true);
+		}
+		new DefaultToolItem(new WorkbenchShell(), CENTRAL_LABEL).click();
+		// activate central editor
+		new DefaultEditor(CENTRAL_LABEL);
+	}
+
+	// https://www.mkyong.com/java/json-simple-example-read-and-write-json
+	private static void loadBlacklistErrorsFile(String blacklisterrorsFile) {
+		if (blacklisterrorsFile.isEmpty()) {
+			return;
+		}
+		String pathToFile = "";
+		try {
+			pathToFile = new File(blacklisterrorsFile).getCanonicalPath();
+			JSONParser parser = new JSONParser();
+			blacklistErrorsFileContents = (JSONObject) parser.parse(new FileReader(pathToFile));
+		} catch (IOException ex) {
+			fail("Blacklist file not found! Path is: " + pathToFile);
+		} catch (ParseException e) {
+			fail("ParseException: unable to parse file at path is: " + pathToFile);
+		}
+	}
+
+	@Test
+	public void createNewLauncherApplicationProjectWithDefaultSettings() {
+		ArchetypeProject project = new org.jboss.tools.central.test.ui.reddeer.projects.LauncherApplication();
+
+		log.step("Import project start");
+		JBossCentralProjectWizardLauncherApplication dialog = new JBossCentralProjectWizardLauncherApplication(project);
```

Comment:
project variable should not be needed to create JBossCentralProjectWizardLauncherApplication - you can delete Archetype project completely from this class and instead archetype project use only constant "Launcher Application" to simplify code

Reply:
Deleted and tested on localhost.

Reply:
Tested on Jenkins, works good.